CodeEditModules/Modules/CodeEditUI/src/PanelDivider.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  PanelDivider.swift
+//  
+//
+//  Created by Austin Condiff on 5/10/22.
+//
+
+import SwiftUI
+
+public struct PanelDivider: View {
+    @Environment(\.colorScheme)
+    private var colorScheme
+
+    public init() {}
+
+    public var body: some View {
+        Rectangle()
+            .foregroundColor(
+                Color(nsColor: .black)
+                    .opacity(colorScheme == .dark ? 0.65 : 0.13)
+
+            )
+            .frame(height: 1.0)
```

Comment:
If used in HStack it should probably be `width: 1.0` to be similar to `Divider`. Should we do this now? Any idea how this might be done?

Reply:
I don't know how to automatically switch behavior when either HStack, or VStack is used but the `body` can be written as simply:

```swift
public var body: some View {
  Color.black 
    .opacity(colorScheme == .dark ? 0.65 : 0.13)
    .frame(height: 1.0)
}
```

Since `Color` is also a view and acts like a `Rectangle` by default.

Reply:
Alternatively you could set the `Divider()` color by using the `background()` modifier:

```swift
Divider()
  .background(Color.black)
```

Reply:
> If used in `HStack` it should probably be `width: 1.0` to be similar to `Divider`. Should we do this now? Any idea how this might be done?

@austincondiff : Do you mean making `PanelDivider` support both dividing horizontally and dividing vertically?
It is possible to do something like this:
```swift
.frame(width: direction == .vertical ? 1.0 : nil, height: direction == .horizontal ? 1.0 : nil)
```

Reply:
The `overlay` approach looks neat! And yeah the `foregroundColor` would not override the one from `Divider()`.